packages/components/src/components/clipboard/clipboard.tsx

```diff
@@ -25,6 +26,7 @@ const Clipboard = ({
     popoverClassName,
     popover_props = {},
     popoverAlignment = 'bottom',
+    size,
```

Comment:
```suggestion
    size = 16,
```
Best to add the default here instead of the component level